RLBotDiscordBot/calendars.py

```diff
@@ -0,0 +1,75 @@
+import requests
+import datetime
+import json
+import logging
+import sys
+import random as r
+from urllib.parse import urlparse
+import re
+import os
+from config import GOOGLE_API_KEY
+import natural
+from natural import date
+
+import discord
+from discord.ext import commands
+from discord.utils import get
+
+
+async def checkCalendar(message):
+    api_key = GOOGLE_API_KEY
+    args = message.content.split(" ")
+    if "!tournaments" in args:
+        months = ["January", "February", "March", "April", "May", "June", "July", "August", "September","October", "November", "December"]
+        FORMAT = "%Y-%m-%dT%H:%M:%SZ"
+        FORMAT2 = "%H:%M UTC"
+        FORMAT3 = "%d %m"
+        today = datetime.datetime.utcnow()
+        to_check = today.strftime(FORMAT)
+        r = requests.get(f"https://www.googleapis.com/calendar/v3/calendars/rlbotofficial@gmail.com/events?maxResults=3&timeMin={to_check}&key={api_key}")
+        jsons = r.json()
+        if len(jsons["items"]) > 0:
+            name = jsons["items"][0]["summary"]
+            start = jsons["items"][0]["start"]["dateTime"]
+            new_date = datetime.datetime.strptime(start, FORMAT)
+            some_time = new_date.strftime(FORMAT2)
+            day_month = new_date.strftime(FORMAT3).split(" ")
+            day = day_month[0]
+            month = months[int(day_month[1])-1]
+            month_day = month + " " + day
+            time_until = date.duration(new_date, now=today, precision=3)
```

Comment:
I noticed this block of code is duplicated three times, you should try to either put it in a loop, or if that's not fully possible, then put it in a function that creates the text. Then it'll be easier to maintain based on the "don't repeat yourself" principle.